app/src/common/translations/en.json

```
@@ -20,7 +20,8 @@
   "language": {
     "headline": "Language",
     "en": "English",
-    "de": "Deutsch"
+    "de": "Deutsch",
+    "fr": "French"
```

To follow best practices, I suggest always displaying the language in the language itself (so "Deutsch" instead of "German" and "Français" instead of "French").

Note: Currently, if a label translation doesn't exist, English will be used as fallback. Hence, it is not necessary to add language-specific translations for words that are the same in English. So the idea is to define the language only in `en.json` (in the language itself) and not override it for any other language (so every language displays the same values).

I will add this information to the development documentation.

```suggestion
    "fr": "Français"
```

---

Ok no problem.